### DISCUSSION TITLE
Sharing postgres or redis image with other services

### DISCUSSION THREAD

**Discussion by Sid127** [nextcloud/all-in-one#5502] [Questions] [answered]
Hello, I plan to run run NC-AIO on a low powered machine that already has a postgres instance running. Would it be possible to reuse that instance with AIO, or possible to replace that local instance with NC-AIO's docker image? Would the same be possible for the redis image?

Thanks in advance!

**Comment by szaimen** [comment, marked as answer]
Hi,

> Sharing postgres or redis image with other services

This is unfortunately not possible.
